packages/dotlottie-js/src/tests/utils-node.spec.ts

```diff
@@ -179,6 +181,23 @@ describe('getAnimation', () => {
 
     expect(animation).toEqual(dotLottieLottie1 as AnimationData);
   });
+
+  it('returns inlined images within the animation', async () => {
+    const animation = await getAnimation(bullAnimation, '679b6de0-4786-4239-b91e-4aa45853cc64', { inlineAssets: true });
+
+    expect(animation?.assets?.length).toEqual(5);
```

Comment:
This assertion is not necessary as in the next line we are asserting on the equality of the assets array